lib/src/ApiInstances.js

```diff
@@ -3,7 +3,7 @@ import ChainWebSocket from "./ChainWebSocket";
 import GrapheneApi from "./GrapheneApi";
 import ChainConfig from "./ChainConfig";
 
-let inst;
+if (global) { global.inst = ""; } else { let inst; };
```

Comment:
I don't understand how this fixes react-native support. `inst` is still used in the rest of the code, but if global exists it won't be defined which should cause issues.

Reply:
in react-native, if the variate is not defined like this `global.xxx=""`,it is just a local scope variate. I think it is only a compatibility issue in react-native, because it works well in both node and browser env.

Reply:
Ok that makes sense, thanks